src/types/types.ts

```diff
@@ -100,6 +100,20 @@ export type ImageToolData<Actions = {}, AdditionalFileData = {}> = {
   } & AdditionalFileData;
 } & (Actions extends Record<string, boolean> ? Actions : {});
 
+/**
+ * @description Tunes that would be available on each image block.
+ */
+export type FeaturesConfig = {
+  /** Flag to enable/disable tune - background. */
+  background: boolean;
+  /** Flag to enable/disable tune - border */
+  border: boolean;
+  /** Flag to enable/disable caption */
+  caption: boolean;
```

Comment:
caption should be `boolean | "optional"`

where 
- `true` — alway display (current behaviour)
- `false` — always hide
- `'optional'` — hide input and add a "Caption" tune which will show the field